api/controller.py

```diff
@@ -212,7 +212,7 @@ def load_settings(self):
         interface.
         """
         LogConfiguration.initialize(self._db)
-        self.analytics = Analytics(self._db)
+        self.analytics = Analytics(self._db, refresh=True)
```

Comment:
Why do we need to set `refresh` to `True` here? Won't it cause multiple initializations?

Reply:
I expect this get run only when we are intentionally reloading. From the logs I produced previously, the biggest impacts were coming from loading here:
```
...from_license_source, /Users/Shared/src/tpp/circulation/core/metadata_layer.py, 112: 
... analytics=Analytics(_db) ...
```
and here
```
ODL2API.__init__, /Users/Shared/src/tpp/circulation/api/odl.py, 221: 
self.analytics = Analytics(_db)
```

Having `refresh=True` here allows us to reload when the configuration changes. We can change that if we decide that this is not desirable.